mycli/main.py

```diff
@@ -799,7 +823,10 @@ def format_output(self, title, cur, headers, expanded=False,
                 formatted = self.formatter.format_output(
                     rows, headers, format_name='vertical')
 
-            output.append(formatted)
+            if isinstance(formatted, (str, text_type)):
```

Comment:
Oh! I think it's fixed by CLI Helpers branch `fix_unicode_problems`. It's related to the standard-library `csv` module in 2.7. It returns bytes instead of unicode, while the [`backports.csv`](https://github.com/ryanhiebert/backports.csv) library emulates Python 3 and uses unicode.

Let's just leave this as is for now.